### PR TITLE
Major Bug Fixes + Revamped

### PR DESCRIPTION
**Pull request by cs-rahul-vishwakarma**
**PMDB ID:** 21472
**Bug Fixes:**
    - Mantis 0869808: Error message changed to invalid credentials
    - Mantis 0869812: Action renamed and tooltips added
    - Mantis 0869814: Param name changed to "Process Response As"
    - Mantis 0869816: Sorting direction fixed
    - Mantis 0869818: Params renamed
    - Mantis 0869820: API Issue: Page Size is limited to 500 only
    - Mantis 0869809, 0869823: Tooltip Added
    - Mantis 0869826: Action's name, param, description changed
    - Mantis 0869838: Splitted into 2 actions - get_report_metadata, get_indicators_of_report
    - Mantis 0869840: Action removed
    - Mantis 0869811: Output schema modified
   
**UTCs:**
    - Connector installation verified.
    - Actions and Playbooks list verified.
    - Actions tested.
    - Output schema verified.
    - All Playbooks are in info and inactive mode verified.

### CODE REVIEW

File: dragos-worldview-threat-intelligence/connector.py [hunk 1]
```diff
@@ -6,10 +6,10 @@
 from connectors.core.connector import Connector, get_logger, ConnectorError
 from .operations import operations, _check_health
 
-logger = get_logger("dragos")
+logger = get_logger("dragos-worldview-threat-intelligence")
 
 
-class DragosConnector(Connector):
+class DragosWorldviewConnector(Connector):
```
Comment (cs-suyog-jinturkar) [inline dragos-worldview-threat-intelligence/connector.py]: name can be "DragosWorldviewThreatIntel"

Comment (cs-rahul-vishwakarma) [reply dragos-worldview-threat-intelligence/connector.py]: Class name changed to "DragosWorldViewThreatIntel"

File: dragos-worldview-threat-intelligence/info.json [hunk 2]
```diff
@@ -322,10 +323,10 @@
             "ASC",
```
Comment (cs-shubham-morankar) [inline dragos-worldview-threat-intelligence/info.json]: Please change the options value to ["Ascending", "Descending"]. Also, make changes wherever you have use Sort Order parameter.
Note: Handle those values in constants.py file

File: dragos-worldview-threat-intelligence/info.json [hunk 3]
```diff
@@ -361,23 +362,24 @@
...
-          "tooltip": "CSV/list of serials",
-          "description": "Specify CSV/list of serial for which products you want to retrieve. Search for products matching a report serial.",
+          "tooltip": "List of report serial numbers",
+          "description": "Specify the list of report serial numbers for which you wish to retrieve the details. Search for reports matching the specified serial(s).",
           "required": false,
           "editable": true,
           "visible": true,
           "type": "text",
           "name": "serials",
-          "placeholder": "serial1, serial2"
```
Comment (cs-shubham-morankar) [inline dragos-worldview-threat-intelligence/info.json:371]: Please make change in placeholder with "For e.g. serial1, serial2, etc".  Also, make changes wherever use placeholder key.

**Review by cs-shubham-morankar — COMMENTED** [review]
Review Done. Please make changes as commented and discuss with you.

File: dragos-worldview-threat-intelligence/info.json (hunk 3 above)
Comment (cs-rahul-vishwakarma) [reply dragos-worldview-threat-intelligence/info.json:371]: Placeholders updated as per recommendation.

File: dragos-worldview-threat-intelligence/info.json (hunk 2 above)
Comment (cs-rahul-vishwakarma) [reply dragos-worldview-threat-intelligence/info.json]: Done.

**Review by cs-shubham-morankar — APPROVED** [review]
Approved.

**Pull request merged**